worlds/mlss/Options.py

```diff
@@ -243,7 +243,7 @@ class HarhallsPants(Toggle):
     This will remove the Harhall's Pants check from the pool.
```

Comment:
This text still says "remove" so it's unclear to me why this is changing to "disable"